tests/functional/aws-node-sdk/test/versioning/objectCopy.js

```diff
@@ -617,7 +620,7 @@ testing('Object Version Copy', () => {
                     Bucket: destBucketName,
                     Key: destObjName,
                     CopySource: `${sourceBucketName}/${sourceObjName}` +
-                    `?versionId=${encodeURIComponent(deleteMarkerId)}`,
+                    `?versionId=${deleteMarkerId}`,
```

Comment:
FYI this should be taken care of by my pr (you may need to rebase once it is merged) : https://github.com/scality/S3/pull/684